Assets/SEE/Controls/Actions/DeleteAction.cs

```diff
@@ -151,8 +196,15 @@ public override void Undo()
         public override void Redo()
         {
             base.Redo();
-            GameElementDeleter.Delete(hitGraphElement);
-            new DeleteNetAction(hitGraphElement.name).Execute();
+            foreach (GameObject go in hitGraphElements)
+            {
+                if (go.IsRoot())
+                {
+                    continue;
+                }
+                _ = GameElementDeleter.Delete(go);
```

Comment:
If this is actually an async method, it should be called `DeleteAsync`, per C# convention. In that case, you can call `.Forget()` on it instead of discarding its result.

Reply:
The method does not look asynchronous at all. I don't know either why the linter warns about an awaitable result.

Reply:
You're right—I really don't get why the IDE would complain here then. I think instead of discarding the results, we should perhaps just disable that VSTHRD110 rule either for this line or for the file instead of discarding the result, it makes the intention a bit more explicit.

Reply:
Looks a bit clumsy but I couldn't find a one-line solution:
```suggestion
                {
                    continue;
                }
                #pragma warning disable VSTHRD110
                GameElementDeleter.Delete(go);
                #pragma warning restore VSTHRD110
```

Reply:
However, I think it would make sense to move the following line up so that it is executed before the delete of `go`:
```
                new DeleteNetAction(go.name).Execute();
```